indico/modules/events/surveys/templates/survey_results.html

```diff
@@ -5,5 +5,146 @@
 {% endblock %}
 
 {% block content %}
-    TODO
+    <div id="survey-results" class="i-box-group vert">
+        {% for question in survey.questions %}
+            <div class="i-box titled">
+                <div class="i-box-header">
+                    <div class="i-box-title">
+                        {{ loop.index }}. {{ question.title }}
+                    </div>
+                    <div class="i-box-metadata">
+                        <span class="label">{% trans %}Answered:{% endtrans %}</span>
+                        <span class="content">{{ question.not_empty_answers|count }}</span>
+                    </div>
+                </div>
+                <div class="i-box-content">
+                    {% if question.field_type == 'text' %}
+                        {{ text_answer(question) }}
+                    {% elif question.field_type == 'number' %}
+                        {{ number_answer(question) }}
+                    {% elif question.field_type in ('bool', 'single_choice') %}
+                        {{ single_choice_answer(question) }}
+                    {% elif question.field_type == 'multiselect' %}
```

Comment:
in that case let's open an issue. it's one of the easy things new developers can do ;)